core/utils/file-helper.js

```diff
@@ -24,4 +24,32 @@ let deleteRecursive = function (path) {
     }
 };
 
-module.exports.deleteFolder = deleteRecursive;
\ No newline at end of file
+module.exports.deleteFolder = deleteRecursive;
+
+/**
+ * Recursively returns all files contained in the specified directory.
+ * @param {string} dir Path to directory.
+ * @param {Array<string>} filelist Currently discovered files list.
+ * @returns {Array<string>} List of files in directory tree.
+ */
+let readRecursive = function (dir, filelist) {
+
+    if (dir[dir.length - 1] !== '/') {
+        dir = dir.concat('/');
+    }
+
+    const files = fs.readdirSync(dir);
+    filelist = filelist || [];
+    files.forEach(function (file) {
+        if (fs.statSync(dir + file).isDirectory()) {
+            filelist = readRecursive(dir + file + '/', filelist);
```

Comment:
I'm surprised eslint isn't enforcing double quotes, we should probably fix that in another PR